_posts/2013-08-10-companies.markdown

```diff
@@ -102,6 +104,8 @@ layout: sidebar
 | `trade_name`              | trade name of this company. This is an optional field.
 | `ein`                     | employer identification number for this company.
 | `entity_type`             | type of company.
+| `is_suspended`            | a boolean representing whether or not the company is suspended in Gusto. A company may choose to suspend their account if they are no longer using Gusto. Gusto may also choose to suspend an account for various reasons.
+| `approval_status`         | a string representing the company's approval status in Gusto. If a company has been approved to use Gusto, the approval status will be "Approved". If a company has not been approved to use Gusto, this field will contain the reasoning.
```

Comment:
Is the reasoning a string? If so, do we reveal anything sensitive or embarrassing for the `approval_status` reason (e.g. not enough money in their bank account)? I wouldn't want to reveal something potentially sensitive or embarrassing. Would it make sense to normalize to "Approved" and "Not Approved"? It may not be that cut and dry though.

Reply:
I was re-reviewing this PR this morning and thinking the same thing because the reasoning can say things like "Rejected due to invoice failure". 

I made a PR to change this back to a boolean in the API: https://github.com/Gusto/zenpayroll/pull/56399

Let me know what you think and if that PR is merged, I'll update this branch.

Reply:
Approved the other PR! Should we explain more about the difference between `is_suspended` and `is_approved`? For example, if a company is approved but suspended, is that confusing? I want to make sure a third party doesn't use the wrong field. :)